primitives/utility/src/lib.rs

```diff
@@ -297,6 +297,43 @@ impl<
 	}
 }
 
+pub struct XcmFeeManagerToAccount<FungiblesMutateAdapter, AccountId, ReceiverAccount>
+where
+	FungiblesMutateAdapter: xcm_executor::traits::TransactAsset,
+{
+	_phantom: PhantomData<(FungiblesMutateAdapter, AccountId, ReceiverAccount)>,
+}
+
+impl<FungiblesMutateAdapter, AccountId, ReceiverAccount> FeeManager
+	for XcmFeeManagerToAccount<FungiblesMutateAdapter, AccountId, ReceiverAccount>
+where
+	FungiblesMutateAdapter: xcm_executor::traits::TransactAsset,
+	AccountId: Clone + Into<[u8; 32]>,
+	ReceiverAccount: frame_support::traits::Get<Option<AccountId>>,
```

Comment:
what about to change `ReceiverAccount: frame_support::traits::Get<Option<MultiLocation>>`?
it will be more general and xcm-like,

and `let dest = X1(AccountId32 { network: None, id: RELAY_TREASURY_PALLET_ID.into_account_truncating() }).into();`  can be configured also in runtime

Reply:
I don't think that's a good idea. And if you really need this feature for whatever reason, just implement it on a new struct.